public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -160,31 +160,113 @@
                     <ul class="drawer__column">
                         <li class="drawer__item">
                             <p class="drawer__item-title">Content type</p>
-                            <wf-editable class="drawer__item-content" ng-model="contentItem.contentTypeTitle" wf-editable-on-update="onBeforeSaveWorkingTitle(newValue)" wf-editable-required="true">
+                            <span class="drawer__item-content">
                                 {{ contentItem.contentTypeTitle || '-' }}
-                            </wf-editable>
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Headline (YT title)</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.title || '-' }}
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Standfirst</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.description || '-' }}
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Trail text</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.trailText || '-' }}
+                            </span>
                         </li>
 
                         <li class="drawer__item">
                             <p class="drawer__item-title">Usages</p>
                             <div class="drawer__item-content" ng-repeat="usage in capiData.usages">
                                 <div>
                                     {{ usage.webTitle || '-' }}
-                                    <a href="{{ usage.composerUrl }}" target="_blank"><i class="drawer__icon" wf-icon="composer"/></a>
-                                    <a href="{{ usage.viewerUrl }}" target="_blank"><i class="drawer__icon" wf-icon="preview"/></a>
-                                    <a href="{{ usage.webUrl }}" target="_blank"><i class="drawer__icon" wf-icon="live-site"/></a>
+                                    <a href="{{ usage.composerUrl }}" target="_blank"><i class="drawer__icon" wf-icon="composer"></i></a>
+                                    <a href="{{ usage.viewerUrl }}" target="_blank"><i class="drawer__icon" wf-icon="preview"></i></a>
+                                    <a href="{{ usage.webUrl }}" target="_blank"><i class="drawer__icon" wf-icon="live-site"></i></a>
                                 </div>
                             </div>
                         </li>
                     </ul>
+
+                    <ul class="drawer__column">
+                        <li class="drawer__item">
```

Comment:
Would it be appropriate to use `ngRepeat` in these drawers? Feels like it might be easier to modify and reason with when we add more content types if this was built up as an array of objects and then looped over?